src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs

```diff
@@ -15,19 +16,29 @@ namespace NuGetConsole
     /// </summary>
     public partial class ConsoleContainer : UserControl
     {
-        public ConsoleContainer(
-            ISolutionManager solutionManager,
-            IProductUpdateService productUpdateService,
-            IPackageRestoreManager packageRestoreManager,
-            IDeleteOnRestartManager deleteOnRestartManager,
-            IVsShell4 shell)
+        public ConsoleContainer(IVsShell4 shell)
         {
 
             InitializeComponent();
 
-            RootLayout.Children.Add(new ProductUpdateBar(productUpdateService));
-            RootLayout.Children.Add(new PackageRestoreBar(solutionManager, packageRestoreManager));
-            RootLayout.Children.Add(new RestartRequestBar(deleteOnRestartManager, shell));
+            ThreadHelper.JoinableTaskFactory.StartOnIdle(
```

Comment:
🥇

Reply:
a brief explanation on what this does would be great for us!

Reply:
Starts it on idle